scripts/run-python-tests.sh

```diff
@@ -10,6 +10,9 @@ set -eu
 scriptdir=$(readlink -f "$(dirname "$0")")
 cd "$scriptdir/.."
 
+. "$scriptdir/util.sh"
+pythoncmd=`getPythonCmd`
+
 # Linux filesystems are case-sensitive, so we need to tell the python unit test
 # scanner to look for upper-case files as well:
-python3 -m unittest discover --verbose --locals --failfast -s source/Mlos.Python -p "[Tt]est*.py"
+$pythoncmd -m unittest discover --verbose --locals --failfast -s source/Mlos.Python -p "[Tt]est*.py"
```

Comment:
I can split this change out to a separate PR if you want.

Reply:
Basically, in one container I tested with 3.8 and 3.7 were both available, but python3 called 3.8 which broke things.

Reply:
I feel like conda might be the better end-game but we don't have to do that now.

Reply:
Might be true.
My intention was to use the Docker build images to run all of the Linux based checks.  The reduces the need to refetch a lot of the pip/apt requirements everytime.